src/script/script_flags.h

```diff
@@ -48,6 +48,10 @@ enum {
     // Note: The Segwit Recovery feature is a (currently moot) exception to
     // VERIFY_INPUT_SIGCHECKS
     SCRIPT_VERIFY_INPUT_SIGCHECKS = (1U << 22),
+
+    // Require BIP341 sighash for key spend path (spending without revealing the
+    // script) in Taproot
+    SCRIPT_REQUIRE_BIP341_SIGHASH = (1U << 23),
```

Comment:
May want to move this down a bit since ABC may use this bit, though it's easy enough to rearrange theirs in a backport.